components/org.wso2.carbon.siddhi.editor.core/src/main/java/org/wso2/carbon/siddhi/editor/core/util/siddhiappdeployer/SiddhiAppApiHelper.java

```diff
@@ -0,0 +1,52 @@
+package org.wso2.carbon.siddhi.editor.core.util.siddhiappdeployer;
+
+import feign.Response;
+import feign.RetryableException;
+import org.wso2.carbon.siddhi.editor.core.exception.SiddhiAppDeployerServiceStubException;
+import org.wso2.carbon.siddhi.editor.core.exception.SiddhiAppsApiHelperException;
+import org.wso2.carbon.siddhi.editor.core.util.siddhiappdeployer.api.SiddhiAppApiHelperService;
+import org.wso2.carbon.siddhi.editor.core.util.siddhiappdeployer.util.HTTPSClientUtil;
+
```

Comment:
Add java doc comments for all the new classes and methods

Reply:
added the changes